tests/unit/test_tracing.py

```diff
@@ -125,9 +125,11 @@ def greeting(name, message):
     # and add its response as trace metadata
     # and use service name as a metadata namespace
     assert in_subsegment_mock.in_subsegment.call_count == 1
-    assert in_subsegment_mock.in_subsegment.call_args == mocker.call(name="## greeting")
+    assert in_subsegment_mock.in_subsegment.call_args == mocker.call(
+        name="## unit.test_tracing.test_tracer_method.<locals>.greeting"
+    )
     assert in_subsegment_mock.put_metadata.call_args == mocker.call(
-        key="greeting response", value=dummy_response, namespace="booking"
+        key="unit.test_tracing.test_tracer_method.<locals>.greeting response", value=dummy_response, namespace="booking"
```

Comment:
make it as a constant and use f-string to make it easier to read/maintain later.

Reply:
Now it's a constant

![image](https://user-images.githubusercontent.com/4295173/188700233-10b9e9ba-5ca8-43c3-a317-40d5cb7a3ea7.png)